### DISCUSSION TITLE
The difference between python@3.11 versions 3.11.2 and 3.11.2_1

### DISCUSSION THREAD

**Discussion by rafrafek** [Homebrew/discussions#4244] [Everyday usage] [answered]
Hi!

What is the difference between python@3.11 versions 3.11.2 and 3.11.2_1? I found this commit https://github.com/Homebrew/homebrew-core/commit/1cc59aee8ca20d08915a4c50c10608ae1fd1b98b but it doesn't help to understand what has changed.

Thanks!

**Comment by p-linnane** [comment, marked as answer]
Here is the relevant PR: https://github.com/Homebrew/homebrew-core/pull/123165

We migrated `python@3.11` to be the default Python for Homebrew. This revision let's us have everyone be on the same bottles (packages) of `python@3.11`, but it's still just 3.11.2.
